### DISCUSSION TITLE
Creating compelling app icons

### DISCUSSION THREAD

**Discussion by micahilbery** [elementary/edw#33] [Accepted Talks]
Speaker: @micahilbery 

In this talk we'll cover how to make good app icons, why a good app icon is important, a brief walkthrough of the HIG (Human Interface Guidelines) surrounding icons, and a demonstration of how to use the these guidelines to make an icon.

**Comment by cassidyjames** [comment]
@micahilbery I'm not sure if you saw, but we've accepted your talk. :) You can get started writing it now if you'd like, and we will provide further instructions/guidance around the recording once we've accepted all the talks.
